<METHOD_START> @ Test public void void ( ) java.lang.Exception { runTestsAndAssertCounters ( SpringRunner .class , org.springframework.test.context.junit4.ExpectedExceptionSpringRunnerTests.ExpectedExceptionSpringRunnerTestCase .class , NUMBER , NUMBER , NUMBER , NUMBER , NUMBER ) ; }  <METHOD_END>
<METHOD_START> @ Test ( expected = java.lang.IndexOutOfBoundsException .class ) public void void ( ) { new java.util.ArrayList<java.lang.Object> <> ( ) . java.lang.Object ( NUMBER ) ; }  <METHOD_END>
